mobify/platform-scaffold
# Integrate Messaging commands

(brief description / PR title)

 **JIRA**: https://mobify.atlassian.net/browse/MSG-482
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/656

## Changes
- Add `scripts` entries for the `sdk-messaging` commands to `web/package.json`

## How to test-drive this PR
- using a link to the `develop` SDK, verify that `npm run messaging:certificate` and `npm run messaging:upload` work, when invoked from the `web` directory. Note that the commands may fail because the Messaging config file isn't named for the project (the generator renames it) - this isn't a failure of this change.
